### DISCUSSION TITLE
[A-L] (2023/24) Foglio 2 - Esercizio 8

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#171] [Esercizi - Viaggi/Piazza]
![Es 8](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/829e5838-c9fe-4850-9e9a-539baa4b4e80)
- Suggerimento in #19

**Comment by Elia-Belli** [comment]
- Siano $a,b \in \mathcal{U}(A) \Rightarrow \exists\ a',b' \in A$ rispettivamente inversi di $a,b$ rispetto a $\cdot$
Sia $a \cdot b$ il loro prodotto, il cui inverso è $(a \cdot b)'$, dobbiamo dimostrare che esso è in $\mathcal{U}(A)$:
per la **(1.3)** sappiamo che $(a \cdot b)'= b' \cdot a'$, per ipotesi $a',b' \in A \Rightarrow b' \cdot a' \in A$
siccome $(a \cdot b)' \in A$ e $(a \cdot b)' \cdot (a \cdot b) = 1= (a \cdot b) \cdot (a \cdot b)' \Rightarrow (a \cdot b) \in \mathcal{U}(A)$.

- Affinchè $(\mathcal{U}(A),\cdot)$ sia un gruppo, dove , deve soddisfare le seguenti proprietà:
$\cdot := \mathcal{U}(A) \times \mathcal{U}(A) \rightarrow \mathcal{U}(A) | \cdot(a,b)\rightarrow (a \cdot b)$
  - **Associatività** : siccome $\cdot$ è associativa per $A$, e $\mathcal{U}(A) \subset A$ allora è ancora associativa in $\mathcal{U}(A)$;
  - **Esistenza dell'elemento neutro**: l'elemento neutro in $A$ era $1$, siccome $\exists a \in A| a \cdot 1 = 1 = 1 \cdot a$, ovvero se stesso, allora $1 \in \mathcal{U}(A)$;
  - **Esistenza dell'elemento inverso**: per definizione dell'insieme $\mathcal{U}(A)$ se $a \in \mathcal{U}(A) \Rightarrow \exists a' \in A$, siccome l'inverso di $a'$ è $a$ già nell'insieme, allora $a' \in \mathcal{U}(A)$.

**Comment by CiottoloMaggico** [comment]
![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/79638284/364f666f-7435-4213-a2f5-df754f76ea7e)

**Comment by AglaiaNorza** [comment]
Soluzione ad opera della tutor Elisa Scandiuzzi, lezione del 23 ottobre 2024 (`appunti_24_10_23.pdf`):

![es](https://github.com/user-attachments/assets/f5765022-6c21-4973-bde1-0258c72b9a70)
